pom.xml

```diff
@@ -87,7 +87,7 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-shade-plugin</artifactId>
-          <version>2.1</version>
+          <version>3.1.0</version>
```

Comment:
I had to bump this dependence; with version 2.1, I got a crash in ASM during the class renaming process.